frontend/app/src/components/history/events/HistoryEventsList.vue

```diff
@@ -128,6 +128,17 @@ watch(
   }
 );
 
+const headerItems = computed(() => {
+  const items: Record<string, any> = {};
+  const blockEvent = get(isEthBlockEventRef(eventGroupHeader));
+
+  if (blockEvent) {
+    items.blockNumber = blockEvent.blockNumber.toString();
+  }
+
+  return items;
+});
+
```

Comment:
Are we going to add more stuff in the items? it seems a bit redundant.

you either return `{ blockNumber: blockEvent.blockNumber.toString() }` or `{}`

Reply:
For now yes it's only blockNumber, just prepared if in the future we need more fields.
Should I just do it directly only for blockNumber for now?
```
blockNumber: eventGroupHeader.blockNumber?.toString() || item.blockNumber
```

Reply:
No need to complicate it, just keep it simple and we can adjust it in the future if the need arises :)

Reply:
updated